.github/workflows/open-pr.yml

```diff
@@ -0,0 +1,35 @@
+name: Open PR
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  version-label-comment:
+    name: 'Add Comments'
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: mshick/add-pr-comment@v2
+        with:
+          message: |
+            To increment version on merge, please add one of `patch` (default), `minor`, or `major` label to this PR.
+
+  add-default-labels:
+    name: 'Add Labels'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/github-script@v6
+        # skip auto patch labeling for dependabot or renovate PRs
+        if: ${{ github.actor != 'dependabot[bot]' && github.actor != 'renovate[bot]' }}
```

Comment:
we could enable them.. the public can't merge PRs onto main, we should be ok on that front

Reply:
for instance, if you or I or Constanza approves a bot PR and we think it's worth a bump, we can label the PR

Reply:
Oh, you mean allow version bumps from the bot PRs? I thought to disable them because we might not want a version for every single dependency upgrade. But i suppose there's no shortage of integers

Reply:
on a semi-related topic, if you think we should use git-commit-sha values instead of npm.. we can discuss that too separately perhaps.

Reply:
>for instance, if you or I or Constanza approves a bot PR and we think it's worth a bump, we can label the PR

That's the workflow I was thinking of—just not applying the label automatically, and making it an intentional action.

Reply:
Bots won't cause version bumps unless the PR has a label.. we should be ok?